pandas/tests/resample/test_resampler_grouper.py

```diff
@@ -347,3 +347,16 @@ def test_median_duplicate_columns():
     result = df.resample("5s").median()
     expected.columns = result.columns
     tm.assert_frame_equal(result, expected)
+
+
+def test_apply_to_one_column_of_df():
+    # GH: 36951
+    df = pd.DataFrame(
```

Comment:
it looks there are 2 examples in the OP, this would address the 2nd, but not the first, is that right?

Reply:
Yes you are right. Missed that one completly. Works now too.